style.css

```diff
@@ -0,0 +1,211 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```

Comment:
You can import your font selecting the different weights and italics you want like this (maybe that's why you couldn't deal with font-weight):

You select your styles by clicking on "Select this style":
<img width="1350" alt="Untitled" src="https://user-images.githubusercontent.com/66212802/95877269-36b2ff80-0d74-11eb-85d2-87d806cd1112.png">

Once you have the ones you like, you can embed them:
<img width="1350" alt="Untitled copy" src="https://user-images.githubusercontent.com/66212802/95877283-3a468680-0d74-11eb-87db-ee8816fef714.png">

This is another way of embedding the font and a better practice, but you'll learn about that later on (although you can do this as you did but with the different styles selected).
<img width="1350" alt="Untitled copy 2" src="https://user-images.githubusercontent.com/66212802/95877294-3ca8e080-0d74-11eb-8d8c-32d186984eb1.png">

Done! :)